kroxylicious-kms-providers/kroxylicious-kms-provider-hashicorp-vault-test-support/src/main/java/io/kroxylicious/kms/provider/hashicorp/vault/AbstractVaultTestKmsFacade.java

```diff
@@ -77,12 +83,46 @@ public final void start() {
         kmsVaultToken = createOrphanToken("kroxylicious_encryption_filter", true, Set.of(policyName));
     }
 
-    protected void enableTransit() {
-        var engine = new EnableEngineRequest("transit");
-        var body = encodeJson(engine);
-        var request = createVaultPost(getVaultUrl().resolve("v1/sys/mounts/transit"), HttpRequest.BodyPublishers.ofString(body));
+    private void withRetry(Runnable r, int maxAttempts) {
```

Comment:
I'm not entirely convinced we need `withRetry` or should be implementing it (rather than using awaitly) but I this is functional and should allow us to make progress.